src/loader/dbconnect.js

```diff
@@ -0,0 +1,16 @@
+const users = [];
```

Comment:
the data should be in json files or database

Reply:
thanks